llvm/test/SYCLLowerIR/printf_generic_as.ll

```diff
@@ -0,0 +1,102 @@
+;; This tests replacement of string literal address space for __spirv_ocl_printf
```

Comment:
I suggest to create a separate subdirectory specifically for the pass, to have better structured tests and shorten their names (`printf_` can be omitted)

Reply:
Done in f78330955bbc9109be0b89f2f69f00e5933b8115